liblwgeom/lwgeom_geos.h

```diff
@@ -19,38 +19,31 @@
  **********************************************************************
  *
  * Copyright 2011 Sandro Santilli <strk@kbt.io>
+ * Copyright 2018 Darafei Praliaskouski <me@komzpa.net>
  *
  **********************************************************************/
 
-
-/* Workaround for GEOS 2.2 compatibility: old geos_c.h does not contain
-   header guards to protect from multiple inclusion */
-#ifndef GEOS_C_INCLUDED
-#define GEOS_C_INCLUDED
 #include "geos_c.h"
-#endif
 
 #include "liblwgeom.h"
 #include "lwunionfind.h"
 
 /*
 ** Public prototypes for GEOS utility functions.
 */
-LWGEOM *GEOS2LWGEOM(const GEOSGeometry *geom, char want3d);
-GEOSGeometry * LWGEOM2GEOS(const LWGEOM *g, int autofix);
-GEOSGeometry * GBOX2GEOS(const GBOX *g);
-GEOSGeometry * LWGEOM_GEOS_buildArea(const GEOSGeometry* geom_in);
+LWGEOM* GEOS2LWGEOM(const GEOSGeometry* geom, uint32_t want3d);
```

Comment:
Why char (one byte) -> uint32_t (4 bytes)?

Reply:
changed to `uint8_t` as it's used as boolean elsewhere in code.